dynamiqs/plots/plots_hinton.py

```diff
@@ -170,49 +191,49 @@ def plot_hinton(
 
         ![plot_hinton_large](/figs-code/plot_hinton_large.png){.fig}
     """  # noqa: E501
+    x = jnp.asarray(x)
```

Comment:
From what I understood, `jnp` can be thought of as "`np` with tracing" so we probably don't need it in plot functions